dags/transportation/dash/trips.py

```diff
@@ -48,10 +48,15 @@
     sqlalchemy.Column("scheduled_depart", sqlalchemy.DateTime(timezone=True)),
     sqlalchemy.Column("stop_href", sqlalchemy.String),
     sqlalchemy.Column("stop_name", sqlalchemy.String),
+    sqlalchemy.Column("trip_id", sqlalchemy.Integer),
     sqlalchemy.Column("trip_href", sqlalchemy.String),
     sqlalchemy.Column("trip_name", sqlalchemy.String),
     sqlalchemy.Column("vehicle_href", sqlalchemy.String),
     sqlalchemy.Column("vehicle_name", sqlalchemy.String),
+    # Add constraints. We want to consider a single stop on a single day
+    # to be unique, so we combine 'trip_id' (which is shared across multiple
+    # days and stops) with 'stop_name' and 'scheduled_arrive'
+    sqlalchemy.UniqueConstraint("scheduled_arrive", "stop_name", "trip_id"),
```

Comment:
Double check directionality.

Reply:
@aclyonsita I think this will work okay, since different directions should have different scheduled arrival times.